.asf.yaml

```
@@ -21,5 +21,8 @@ github:
   features:
     issues: true
 
+notifications:
+  issues_status: issues@arrow.apache.org
```

Could you also add `issues_comment` and `pullrequests`?

```suggestion
  issues_status: issues@arrow.apache.org
  issues_comment: github@arrow.apache.org
  pullrequests: github@arrow.apache.org
```

For commits, we may be able to `commits_by_path`: https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features#Git.asf.yamlfeatures-by-pathcommitemails